docs/source/release/v6.6.0/Diffraction/Engineering/Bugfixes/34331.rst

```diff
@@ -1,2 +1,3 @@
 - Correct the tabbing order between widgets on the Engineering Diffraction interface.
-- Add a scrollbar to the GSASII tab to allow a smaller interface height, like the fitting tab.
\ No newline at end of file
+- Add a scrollbar to the GSASII tab to allow a smaller interface height, like the fitting tab.
+- SaveVulcanGSS algorithm marked as deprecated as of 2022-11-30
```

Comment:
This should be in a new release note, it can be named after the pr since there is no issue `34763.rst`.